packages/firestore/lite/src/api/database.ts

```diff
@@ -93,10 +93,14 @@ export class Firestore implements firestore.FirebaseFirestore {
       this._datastorePromise = PlatformSupport.getPlatform()
         .loadConnection(databaseInfo)
         .then(connection => {
-          const serializer = PlatformSupport.getPlatform().newSerializer(
+          const serializationOptions = PlatformSupport.getPlatform().newSerializationOptions(
```

Comment:
In tree-shakeable style, the structure containing the data we pass through that used to be a class is now just a dumb struct, but it still has the same identity. All that's changed is that it's a dumb struct with accompanying functions instead of a class with member functions.

For parity with the other platforms I think it still makes sense to call that thing a `Serializer`.

Reply:
The parity argument makes sense. I updated the PR accordingly.